src/cmd/init.rs

```diff
@@ -42,8 +45,18 @@ pub fn run_init(
         n_words,
         from_existing,
         xpub,
+        coldcard_sd_path,
     }: InitOpt,
 ) -> anyhow::Result<CmdOutput> {
+    if wallet_dir.exists() {
```

Comment:
Moved from below, so wallet dir is created before attempting to write seed words.